code/datums/mood_events/drink_events.dm

```diff
@@ -26,3 +26,50 @@
 	description = "<span class='nicegreen'>Amazing taste!</span>\n"
 	mood_change = 50
 	timeout = 10 MINUTES
+	
+/datum/reagent/consumable/pilk
```

Comment:
pretty sure you meant to copypaste the mood_events and not reagents here

Reply:
God I'm a fucking moron lmao